### DISCUSSION TITLE
Git Updater headers

### DISCUSSION THREAD

**Discussion by afragen** [tailpress/tailpress#210] [Ideas]
As the `GitHub Theme URI` header was recently added. The theme should be _seen_ by Git Updater. 

Since you’ve change the default branch from `master` to `main` you will also need to add the `Primary Branch: main` as a header.

**Comment by jeffreyvr** [comment]
Not sure if I understand. The default branch of this repo is master. Am I missing something?

**Reply by afragen** [reply]
When I look at the repo, https://github.com/24Degrees/writehistory, the default branch is now "main"

**Reply by afragen** [reply]
@jeffreyvr I think someone forked your repo and the issues still refer back to your repo and not their fork. 

Please disregard.
